modules/service/src/main/resources/graphql/itc.graphql

```diff
@@ -3385,6 +3385,7 @@ input UnnormalizedSedInput {
   powerLaw: BigDecimal
   blackBodyTempK: PosInt
   fluxDensities: [FluxDensity!]
+  fluxDensitiesUrl: String
```

Comment:
This will probably not be the final form, it's more of a PoC for the moment.

Reply:
Is it so that you pass either the array of densities or an URL?